web/css/styles.scss

```diff
@@ -32,7 +32,6 @@
 @import 'blocks/gallery';
 
 // Theme extends
```

Comment:
Could you remove this line too?

BTW. It's looks like duplicated `@import` issue might be related to #34 - thanks for this fix!